packages/cli/templates/vue3/src/utils/convertToRaw.ts

```diff
@@ -0,0 +1,24 @@
+import { toRaw, isProxy, isRef, unref } from 'vue'
+
+export function convertToRaw(value: any) {
```

Comment:
This is how I did it. Probably inefficient, but easy to read:

```
const convertToRaw = (value: unknown) => JSON.parse(JSON.stringify(value))
```

I was hoping that Vue would provide such a function, but I was unable to find it.

Reply:
unfortunately we can't
![Screenshot 2023-04-17 at 16 05 53](https://user-images.githubusercontent.com/499898/232508370-2f89360a-ec53-4037-b16a-a9632a3d36e5.png)